Formula/eless.rb

```diff
@@ -0,0 +1,23 @@
+class Eless < Formula
+  desc "Better 'less' using Emacs view-mode and Bash"
+  homepage "https://eless.scripter.co"
+  url "https://github.com/kaushalmodi/eless/archive/v0.5.tar.gz"
+  sha256 "b4da2c7c223996681bb951d50e0d542d8df04baf765c16412fa0848cdb2b3a3d"
+
+  head do
+    url "https://github.com/kaushalmodi/eless.git"
+  end
+
+  depends_on "emacs"
+  depends_on "bash"
```

Comment:
Both `bash` and `perl` are provided by the system. Are there actually benefits to forcing the latest/greatest versions here?

Reply:
Hello, I am the eless utility developer. It's tested to work in Perl 5.16.3 (RHEL 6.6, my machine) and bash 3.2 (as reported by a macOS user). If those versions are old enough for a majority of systems, I believe that enforcing of the latest versions can be skipped.